recipes/themysto/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# this is a conda-forge recipe. See https://github.com/conda-forge for details
+
+{% set version = "0.0.4" %}
+
+package:
+  name: themysto
```

Comment:
similar to calisto -> calysto as jupiter -> jupiter, see https://en.wikipedia.org/wiki/Themisto for the character.